src/aselo-webchat.ts

```diff
@@ -83,7 +74,20 @@ const setChannelAfterStartEngagement = doWithChannel((channel: Channel, manager:
   channel.sendMessage(translations[initialLanguage].AutoFirstMessage);
 })
 
-export const initWebchat = () => FlexWebChat.createWebChat(appConfig).then(webchat => {
+export const initWebchat = async () => {
+  const ip = await getUserIp();
```

Comment:
Either now or later, we should set IP collection to be configurable.  Some helplines want it, but some are very opposed to it.

Reply:
I can do that now. Do you know if both ZA and ZM want it?

Reply:
Raised that question in slack.

Reply:
I've updated the code so each helpline have to set `captureIp` to `true` or `false`